daft/expressions/expressions.py

```diff
@@ -944,6 +944,20 @@ def log1p(self) -> Expression:
         f = native.get_function_from_registry("log1p")
         return Expression._from_pyexpr(f(self._expr))
 
+    def pow(self, exp: float) -> Expression:  # type: ignore
```

Comment:
could we also add the `__pow__` dunder method, that way users can do `col("x") ** n`